.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
Being EOL doesn't mean you have to drop 5.4. Since in the HTTP interfaces nothing indicates that you need 5.5, I would say try to test against 5.4 as well, even if it means you need a lower version. I would do the following: If the travis php version is 5.4 then remove guzzle6-adapter and install guzzle5-adapter. The guesser will always find the class for you. This way you can test against 5.4. See how it is done in [php-http/discovery](https://github.com/php-http/discovery/blob/master/.travis.yml#L23)